.circleci/extended/orb-test-matrix.yml

```diff
@@ -0,0 +1,12 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1
+
+workflows:
+  sample:
+    jobs:
+      - node/test:
+          matrix:
+            parameters:
+              version: ["15.1", "lts", "12.21"]
```

Comment:
Would be useful to add a comment here pointing users to the other parameters  they can run matrix jobs against. Maybe we add these links here:
https://circleci.com/developer/orbs/orb/circleci/node#jobs-test
https://circleci.com/blog/circleci-matrix-jobs/